code/modules/library/skill_learning/job_skillchips/roboticist.dm

```diff
@@ -0,0 +1,13 @@
+/obj/item/skillchip/roboticist
+	name = "Cyborg C1-RCU-1T skillchip"
+	desc = "A roboticist's second best friend."
+	auto_trait = TRAIT_KNOW_CYBORG_WIRES
+	skill_name = "Cyborg Circuitry"
+	skill_description = "Recognise cyborg wire layouts and understand their functionality at a glance."
+	skill_icon = "sitemap"
+	implanting_message = "<span class='notice'>You suddenly comprehend the secrets behind cyborg circuitry.</span>"
+	removal_message = "<span class='notice'>Cyborg circuitry stops making sense as images of coloured wires fade from your mind.</span>"
+
+/obj/item/skillchip/roboticist/Initialize()
+	. = ..()
+	skillchip_flags |= SKILLCHIP_JOB_TYPE
```

Comment:
You can add this to the object definition, it doesn't need to be on `Initialize()`
Just know beforehand which are the appropriate bitflags.